packages/website/public/llms/styleGuide.md

```diff
@@ -0,0 +1,417 @@
+# Squiggle Style Guide
+
+## Limitations
+
+- There are floating point errors at high numbers (1e50 and above) and very small numbers (1e-10 and below). If you need to work with these, use logarithms if possible.
+
+## Data and Calculations
+
+### Estimations
+
+- When using the "to" format, like "3 to 10", remember that this represents the 5th and 95th percentile. This is a very large range. Be paranoid about being overconfident and too narrow in your estimates.
+- One good technique, when you think there's a chance that you might be very wrong about a variable, is to use a mixture that contains a very wide distribution. For example, `mx([300 to 400, 50 to 5000], [0.9, 0.1])`, or `mx([50k to 60k, 1k to 1M], [0.95, 0.05])`. This way if you are caught by surprise, the wide distribution will still give you a reasonable outcome.
```

Comment:
These are good

Reply:
FYI though, I think the LLM is poor at doing this now, we definitely have more work to do here.